taipy_theme/assets/stylesheets/avaiga.css

```diff
@@ -81,6 +81,12 @@
   --md-hue: 270; 
 }
 
+@media screen and (max-width: 59.9375em){
```

Comment:
59.9375, seriously?

Reply:
Yeah. I dont like it either but that's a 959px on a 16px base, and that's the way it is done in Material Mkdocs CSS...
I personnaly prefer `@media screen and (max-width: calc(60em - 1px))` but hey... Coding consistency. 😔🔫